Source/SelfService/Web/admin/application/view.tsx

```diff
@@ -100,15 +101,15 @@ export const View: React.FunctionComponent<any> = (props) => {
 
     return (
         <>
-            <h1>Customer {customer.name}</h1>
+            <Typography variant='h1' my={2}>Customer {customer.name}</Typography>
```

Comment:
To give an example on how to extend the first comment in app.tsx: 
- designers and product owners dont care about in-depth knowledge on the workings of typography, but, would like to be able to refer to a component that does what they expect it to do. In this case it would be a h1 element with margin on the y axis 2. These are inputs they can reuse in Figma
-  They can create their own templates with their own naming. Since this line:
`<Typography variant='h1' my={2}>`
is reused, i wonder if it can be wrapped in a component which shares the name designers could reuse in Figma and behaves based on this. Not saying this should be done in this PR specifically, but its the opportunity i spot in this PR.

Reply:
If ALL Typography header components use `my={2}` then i suggest to move it up to the theme level. That could be a an improvement done in this PR i find

Reply:
Since marginY is not a css property and actually consists of the following:
<img width="535" alt="Screenshot 2022-05-16 at 11 27 26" src="https://user-images.githubusercontent.com/80390782/168562385-3075b0f3-b2bb-4895-a59f-0796f72d8a2a.png">

You could solve that by setting the css properties in the screenshots to match the expected spacing 
further docs on spacing using themes: https://mui.com/system/spacing/

Reply:
I could have been a bit more clear on the "why" of using the `my={2}`- approach. Basically my idea is that this is a drop-in replacement for all headers currently in the app to lean on Typography element to pick up on the theme's font sizing. Without the Typography element, the styles were being pulled from the browser style - ie, with margin top and bottom for each header tag. By default MUI's Typography has no spacing, so I made a judgement call of 3 spacing elements not to break all the pages where headers were used.

I didn't want to spend time adjusting and amending each implementation, as some pages don't work, others are semi-working, and most are not styled correcyl. Nor am I interested in introducing a global `Title` - element at this time, because I don't consider the current stylings to be the source of truth from a component perspective. Moving forward when implementing more from Figma, I foresee these `Typography` elements to be replaced, and inline-styling them makes that easy to do on a case by case basis.

What I did do was verify one thing, which was cool. even though there is quite a bit of duplication in code, this translates to only 3 CSS rules generated across the app - which is an optimization that is done during the build ( I am unclear where it is happening though)